std/experimental/logger/package.d

```diff
@@ -4,9 +4,8 @@ Implements logging facilities.
 Copyright: Copyright Robert "burner" Schadek 2013 --
 License: <a href="http://www.boost.org/LICENSE_1_0.txt">Boost License 1.0</a>.
 Authors: $(WEB http://www.svs.informatik.uni-oldenburg.de/60865.html, Robert burner Schadek)
-
-$(H3 Basic Logging)
-
+$(H3 Basic Logging
+)
```

Comment:
I know this is ugly and better ideas to fix ddoc are welcome.
Please have a look at the rendered output:
https://dlang.org/phobos/std_experimental_logger.html